app/member-reminders/member-reminders-public/src/main/kotlin/com/hedvig/android/memberreminders/GetNeedsCoInsuredInfoRemindersUseCase.kt

```diff
@@ -40,8 +40,7 @@ internal class GetNeedsCoInsuredInfoRemindersUseCaseImpl(
   }
 
   private fun NeedsCoInsuredInfoReminderQuery.Data.CurrentMember.ActiveContract.hasMissingInfo() =
-    (currentAgreement.coInsured?.filter { it.hasMissingInfo }?.size ?: 0) > 0 ||
-      (upcomingChangedAgreement?.coInsured?.filter { it.hasMissingInfo }?.size ?: 0) > 0
+    (coInsured?.filter { it.hasMissingInfo }?.size ?: 0) > 0 || (coInsured?.filter { it.hasMissingInfo }?.size ?: 0) > 0
```

Comment:
Are we checking the same `hasMissingInfo` on `conInsured` on both of the two || branches here?
Also, if the intention is to check if there is any single entry which has some missing info, could we instead do:
```suggestion
    coInsured?.any { it.hasMissingInfo } == true
```

Reply:
Oh yeah, not sure what happened here...